### DISCUSSION TITLE
Can any one tell me how to use new Data Card Encoder?

### DISCUSSION THREAD

**Discussion by KiritoKing** [GabrielOlvH/Industrial-Revolution#438] [Q&A] [answered]
I has been confused about the new Data Card system for a couple of days. After I upgraded to 1.18.2, I'm unable to get my mining system working and I have no idea how to solve it.
I've tried putting ore blocks into data column but nothing happened.

**Comment by terminhell** [comment, marked as answer]
I think ive sorta figured it out. Put the card in the encoder, add a stack of enriched nikolite dust and a stack of stone on the right 3 spots. Then it seems you need to add an entire stack of the ore you want added to it. Then click Write. The items are consumed and you should see a countdown in the UI indicating somethings happening. Then you'll see some stats on the card when moused over.  As you add more different types of ore to the same card you may need to add either more stone and or enriched nikolite dust. 

Not sure if having to have an entire stack of a particular ore is intended or not.

**Reply by KiritoKing** [reply]
Yep, u are completely right!
I've been testing the Data Card Encoder for the whole afternoon and got that an entire stack is required that my problem lies in.
Besides, other items can be used to modify this process, since redstone dust can accelerate it and bluestone dust can enrich it, but only stones can add the size of the card.

**Reply by terminhell** [reply]
Too bad the in game book doesn't mention any of this. Requiring an entire stack of ore blocks seems like either an over site or just wonky expectations. Sure iron and copper would be easy enough. But diamonds, emeralds, tungsten etc seems a bit excessive. I guess an argument of balance could be made, since this setup basically prints ore in return.
